docs/alerts/scheduled-searches/faq.md

```diff
@@ -93,7 +93,6 @@ Failures can be seen across many Scheduled Searches due to backend infrastructur
 Scheduled Search failures could be the result of other factors, such as a spike in data being processed by the Scheduled Search. 
 
 * If you have a query that normally runs, but is close to reaching the [timeout period limit](#how-to-prevent-your-scheduled-search-from-timing-out), it could finally fail due to changes in your account, such as the content and size of log messages, or the volume or number of messages sent during a specific time range. Choosing a narrower time range can help.
-* You have a lot of scheduled searches running today. 6,000 per person is the maximum.
 * Even if the query normally runs well within the timeout period limit, there a data surge could cause the need for more processing time
 * There could be normal growth factors within an account such as increased Collector deployment that would cause some Source Categories to have a data surge. You should test your Scheduled Searches for performance periodically to address this possibility.
```

Comment:
```suggestion
* There could be normal growth factors within an account, such as increased Collector deployment, that would cause some Source Categories to have a data surge. You should test your Scheduled Searches for performance periodically to address this possibility.
```